### PR TITLE
redshift tests

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -260,21 +262,25 @@ task extractSchema(type: Sync) {
...
-               "cleanReporting": "Cleans the schemas that are required for running locally",
-               "migrateReporting_test": "Migrates the schemas that integration tests depend on",
-               "cleanReporting_test": "Cleans the schemas that integration tests depend on",
-               "migrateWarehouse": "Migrates the schemas that are required for running locally",
-               "cleanWarehouse": "Cleans the schemas that are required for running locally",
-               "migrateWarehouse_test": "Migrates the schemas that integration tests depend on",
-               "cleanWarehouse_test": "Cleans the schemas that integration tests depend on"]
+def taskMap = [
+//        "cleanReporting": "Cleans the schemas that are required for running locally",
+//        "migrateReporting": "Migrates the schemas that are required for running locally",
```
Comment (malaffoon) [inline build.gradle]: Anybody use these? The tasks are `clean/migrateReportingProd`.

File: aggregate-service/build.gradle [hunk 2]
```diff
@@ -33,10 +33,35 @@ dependencies {
...
+// To run these tests locally, you must either manually adjust the application-redshift.yml file
+// (but don't check in your changes!) or set environment variables. If you are using the default
+// CI data resources you should set:
+//   APP_DATAMART_OLAP_DATASOURCE_PASSWORD
+// If you are using different instances the full set of variables can be found in application-redshift.yml.
+// If you are running from the command-line you can use local bash export to do it in one command, e.g.:
+/*
+(export APP_DATAMART_OLAP_DATASOURCE_PASSWORD=secretkey; \
+ gradle rst)
+*/
```
Comment (esotrope) [inline aggregate-service/build.gradle]: Thanks for these comments - very helpful

File: build.gradle (hunk 1 above)
Comment (esotrope) [reply build.gradle]: i use them every so often. what are we supposed to use instead?

Comment (malaffoon) [reply build.gradle]: Oh, i can put them back then. The only real alternative is to run the similar tasks from RDW_Schema, but then the version of RDW_Schema might not match.

**Pull request merged**